src/locale/misc/views/en.yml

```diff
@@ -43,3 +43,4 @@ dataTableErrors:
     modify_column: There was an error editing the column
 suggested:
     created: Created by {name}
+    section_title: Suggested
```

Comment:
This should be camel case. I realize this file is confusing with regards to formatting, but in general camel case should be used for these.

The only time we don't use camel case is if it's a constant from the code (e.g. coming from the backend originally), but this is just a name.